src/electron/main/main.ts

```diff
@@ -11,14 +13,21 @@ import { mainWindowConfig } from './main-window-config';
 let mainWindow: BrowserWindow;
 const platformInfo = new PlatformInfo(process);
 
+const config = new FileSystemConfiguration();
+
 log.transports.file.level = 'info';
 autoUpdater.logger = log;
 
 let recurringUpdateCheck;
 const electronAutoUpdateCheck = new AutoUpdaterClient(autoUpdater);
 
+const os = platformInfo.getOs();
+
+const iconBaseName = path.join(__dirname, '..', config.getOption('electronIconBaseName'));
```

Comment:
I wonder if we should extract this bit out to a separate file; it's pretty small but also pretty independent/testable